.github/workflows/root-ci-config/build_root.py

```diff
@@ -216,6 +218,27 @@ def download_artifacts(obj_prefix: str, shell_log: str):
     return shell_log
 
 
+@github_log_group("Node state")
+def show_node_state(shell_log: str, options: str) -> str:
+    result, shell_log = subprocess_with_log(f"""
+        which cmake
+        cmake --version
+        which c++ || true
+        c++ --version || true
+        uname -a || true
+        cat /etc/os-release || true
+        sw_vers || true
+        uptime || true
+        df || true
+    """, shell_log)
+
+    if result != 0:
+        print_warning("Failed to extract node state")
+
+    shell_log += f"\nBUILD OPTIONS: {options}"
```

Comment:
This is a really nice feature. Could also be printed while running

Reply:
I moved it to `build()`